client/my-sites/checkout/checkout-thank-you/redesign-v2/sections/Footer.tsx

```diff
@@ -1,8 +1,10 @@
+import { isBulkDomainTransfer } from '../../utils';
 import BulkDomainTransferFooter from './footer/BulkDomainTransferFooter';
 import PlanFooter from './footer/PlanFooter';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
 
-const Footer = ( { isBulkDomainFlow }: { isBulkDomainFlow: boolean } ) => {
-	if ( isBulkDomainFlow ) {
+const Footer = ( { purchases }: { purchases: ReceiptPurchase[] } ) => {
+	if ( isBulkDomainTransfer( purchases ) ) {
```

Comment:
Yea, this is what I was thinking how we'd end up doing this when we (explorers) talked about this a little while ago, not sure how much I like it after seeing it though :smile: